src/Config/Level/CodeQualityLevel.php

```diff
@@ -0,0 +1,216 @@
+        JoinStringConcatRector::class,
+        ConsecutiveNullCompareReturnsToNullCoalesceQueueRector::class,
+        ExplicitBoolCompareRector::class,
+        CombineIfRector::class,
+        UseIdenticalOverEqualWithSameTypeRector::class,
+        SimplifyBoolIdenticalTrueRector::class,
+        SimplifyRegexPatternRector::class,
+        BooleanNotIdenticalToNotIdenticalRector::class,
+        CallableThisArrayToAnonymousFunctionRector::class,
+        AndAssignsToSeparateLinesRector::class,
+        CompactToVariablesRector::class,
+        CompleteDynamicPropertiesRector::class,
+        IsAWithStringWithThirdArgumentRector::class,
+        StrlenZeroToIdenticalEmptyStringRector::class,
+        ThrowWithPreviousExceptionRector::class,
+        RemoveSoleValueSprintfRector::class,
+        ShortenElseIfRector::class,
+        ExplicitReturnNullRector::class,
+        ArrayMergeOfNonArraysToSimpleArrayRector::class,
+        ArrayKeyExistsTernaryThenValueToCoalescingRector::class,
+        AbsolutizeRequireAndIncludePathRector::class,
+        ChangeArrayPushToArrayAssignRector::class,
+        ForRepeatedCountToOwnVariableRector::class,
+        ForeachItemsAssignToEmptyArrayToAssignRector::class,
+        InlineIfToExplicitIfRector::class,
+        UnusedForeachValueToArrayKeysRector::class,
+        CommonNotEqualRector::class,
+        SetTypeToCastRector::class,
+        LogicalToBooleanRector::class,
+        VarToPublicPropertyRector::class,
+        IssetOnPropertyObjectToPropertyExistsRector::class,
+        NewStaticToNewSelfRector::class,
+        UnwrapSprintfOneArgumentRector::class,
+        SwitchNegatedTernaryRector::class,
+        SingularSwitchToIfRector::class,
+        SimplifyIfNullableReturnRector::class,
+        FuncGetArgsToVariadicParamRector::class,
+        CallUserFuncToMethodCallRector::class,
+        CallUserFuncWithArrowFunctionToInlineRector::class,
+        CountArrayToEmptyArrayComparisonRector::class,
+        FlipTypeControlToUseExclusiveTypeRector::class,
+        InlineArrayReturnAssignRector::class,
+        InlineIsAInstanceOfRector::class,
+        TernaryFalseExpressionToIfRector::class,
+        InlineConstructorDefaultToPropertyRector::class,
+        TernaryEmptyArrayArrayDimFetchToCoalesceRector::class,
+        OptionalParametersAfterRequiredRector::class,
+        SimplifyEmptyCheckOnEmptyArrayRector::class,
+        SwitchTrueToIfRector::class,
+        CleanupUnneededNullsafeOperatorRector::class,
+        DisallowedEmptyRuleFixerRector::class,
+        ConvertStaticPrivateConstantToSelfRector::class,
+        LocallyCalledStaticMethodToNonStaticRector::class,
+        NumberCompareToMaxFuncCallRector::class,
+        CompleteMissingIfElseBracketRector::class,
+        RemoveUselessIsObjectCheckRector::class,
+        StaticToSelfStaticMethodCallOnFinalClassRector::class,
+        GetClassToInstanceOfRector::class,
+    ];
+
+    /**
+     * @var array<class-string, mixed[]>
```

Comment:
`var array<class-string<RectorInterface>, mixed[]>` should can be used

Reply:
Updated